baseplate/server/__init__.py

```diff
@@ -432,23 +445,90 @@ def load_and_run_shell() -> None:
             from IPython import Config
 
         ipython_config = Config()
+        ipython_config.InteractiveShellApp.exec_lines = [
+            # monkeypatch IPython's log-write() to enable formatted input logging, copying original code:
+            # https://github.com/ipython/ipython/blob/a54bf00feb5182fa821bd5457897b3b30a313436/IPython/core/logger.py#L187-L201
+            f"""
+            ip = get_ipython()
+            from functools import partial
+            ip.magic('logstart {console_logpath}')
+            def log_write(self, data, kind="input"):
+                import time, os
+                if self.log_active and data:
+                    write = self.logfile.write
+                    if kind=='input':
+                        if self.timestamp:
+                            write(time.strftime('# %a, %d %b %Y %H:%M:%S\\n', time.localtime()))
```

Comment:
It might be nice to add more to the log line prefix. It's not clear after the first message where the log is coming from. Even just saying "shell" somewhere in there would be useful (but if we can get the user that shelled in that'd be even better).

Reply:
agreed, and actually took the chance to format modern syslog format across both. this'll persist the pid and hostname across the log lines. also removed the `self.timestamp` code which is an ipython setting, but we're not going to honor it. 

unfortunately we can't get the human user that shelled in in k8s space because of the abstraction with k8s api server. but this'll be correlation done between kube api server logs and these stdout logs